### PR TITLE
Move documentation to carvel.dev

### PR DESCRIPTION
**Pull request by joaopapereira**
- Remove all docs
- Add link to the website documentation page

Related to vmware-tanzu/carvel.dev#92

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,11 +1,3 @@
...
-- [Why](why.md)
-- [Install](install.md)
-- [Walkthrough](walkthrough.md)
-- [Config](config.md)
-- [App CRD spec](app-spec.md)
-- [Security model](security-model.md)
-- [Mozilla's sops integration](sops.md)
-- [Dangerous Flags](dangerous-flags.md)
-- [examples/ directory](../examples/)
+The documentation can be found in https://carvel.dev/kapp-controller/docs/latest/.
```
Comment (danielhelfand) [inline docs/README.md]:
```suggestion
The documentation can be found at https://carvel.dev/kapp-controller/docs/latest/.
```

**Review by danielhelfand — COMMENTED** [review]
Might also be worthwhile to add note about where to contribute documentation changes (i.e. just https://github.com/vmware-tanzu/carvel.dev).

**Comment by joaopapereira** [comment]
@danielhelfand done

**Review by danielhelfand — COMMENTED** [review]
Thanks! lgtm

**Pull request merged**